hack/observability/prometheus/resources/bundle.yaml

```diff
@@ -4486,7 +4486,7 @@ spec:
                               If underlying persistent volume is being resized then
                               the Condition will be set to 'ResizeStarted'.
                             items:
-                              description: PersistentVolumeClaimCondition contails
+                              description: PersistentVolumeClaimCondition contains
```

Comment:
This change is to a YAML resource downloaded by a script, so it could be overwritten. But:
- the script hasn't been run in a long time; we're ok with prometheus staying right here
- if the script is run, the subsequent code change will presumably fail as a PR and not get merged (the current version of this prometheus bundle retains the same spelling error, I checked)